app/views/receipts/_form.html.erb

```diff
@@ -316,17 +358,113 @@
         });
       });
 
-      $('#submit-receipt').click(function (e) {
+      $('#submit-receipt-and-new').click(function (e) {
 
         if ($('.receipt-line').length === 0) {
-          toastr.error("You need to atleast add one commodity.");
+          toastr.error("You need to at least add one commodity.");
```

Comment:
@dawitd rephrase error message to -> "You are required to add at one commodity to receive"